webui/react/package.json

```diff
@@ -21,6 +21,7 @@
     "antd": "^4.6.6",
     "axios": "^0.21.1",
     "can-ndjson-stream": "^1.0.2",
+    "dayjs": "^1.10.4",
```

Comment:
this is needed by `LogViewerTimestamp.tsx` but now needs to be explicitly declared since the migration to `react-scripts 4.0.x`